audb/core/publish.py

```diff
@@ -310,6 +312,15 @@ def publish(
     # load database from folder
     db = audformat.Database.load(db_root)
 
+    if not db.is_portable:
+        raise RuntimeError(
+            f"The database '{db.name}' employs file paths "
```

Comment:
Maybe simplify to: `Some files in the tables have absolute paths...`

Reply:
Changed to:

```python
        raise RuntimeError(
            'Some files in the tables have absolute paths '
            'or use '.' or '..' to address a folder. '
            'Please replace those paths by relative paths '
            'and use folder names instead of dots.'
        )
```

I think, I should spend a little bit more time again when writing those error messages, so you don't have to fix every single one for me ;)